### DISCUSSION TITLE
Welcome to MicSwitch Discussions!

### DISCUSSION THREAD

**Discussion by iXab3r** [iXab3r/MicSwitch#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NikolaKrstic94** [comment]
For Zoom application, when I unmute my microphone with spacebar on ZOOM and my Hotkey forMicSwitch is set to "F9" btw, ZOOM also unmutes my MicSwitch. Please check that.

**Comment by ileathan** [comment]
Rally like this program :).

**Comment by leav6** [comment]
Scott here from Eastern United States. 
I use GitHub Enterprise at work and am interested in becoming involved with the public GitHub community 😁
